RecoParticleFlow/PFClusterProducer/src/GenericSimClusterMapper.h

```diff
@@ -0,0 +1,38 @@
+#ifndef __GenericSimClusterMapper_H__
+#define __GenericSimClusterMapper_H__
```

Comment:
please use more specified include guard (include the  RecoParticleFlow_PFClusterProducer)